android/assets/jsons/translations/German.properties

```diff
@@ -106,8 +106,8 @@ We have encountered the City-State of [name]! = Wir haben den Stadtstaat [name]
 Declare Friendship ([numberOfTurns] turns) = Freundschaft erklären ([numberOfTurns] Runden)
 May our nations forever remain united! = Mögen unsere Nationen auf immer in Freundschaft vereint sein!
 Indeed! = Auf jeden Fall!
-Denounce [civName]? = [civName] anprangern?
-Denounce ([numberOfTurns] turns) = Anprangern ([numberOfTurns] Runden)
+Denounce [civName]? = [civName] denunzieren?
```

Comment:
"Denunzieren" ist zwar die direkte Übersetzung von "Denounce" aber ich fand "anprangern" besser, da es sich flüssiger liest und besser zum Geschehen in Unciv passt.